packages/@aws-cdk/aws-ec2/test/test.vpc.ts

```diff
@@ -666,7 +666,24 @@ export = {
 
       test.done();
     },
+
+    'selecting default subnets in a VPC with only public subnets'(test: Test) {
```

Comment:
Would be nice to also have tests covering:
- selecting "default" subnet IDs returns private subnets IDs
- selecting "Public" subnet IDs returns public subnet IDs
- selecting "named X" subnet IDs returns subnet IDs named X

Now I feel we're only covering the default behavior, which means if there was a bug in the method that reifies the defaults (which would always override what you passed with the defaults), we wouldn't know.

I would also be tempted to nest the tests fo `subnetIDs` under a different test namespace, so that their descriptions are effectively disambiguated from those of `subnets`.

Reply:
Will add the test on the defaults.

The other ones are already tested.